### PR TITLE
Display Case Front Cover that supports SSD1306 Display

### PR DESCRIPTION
**Pull request by Lukas-Sturm**
Display Case Front Cover that supports SSD1306 Display

#### Which mods/configurations/profiles are added by this PR?
- Display Case Front Cover that supports SSD1306 Display and two G-Code Buttons

#### How was it tested? 
- Printed and installed on my voron trident

#### Screenshots (if appropriate)
printed

#### Further notes
only 99% sure that the 2.4 uses the same skirt as the trident

### CODE REVIEW

File: printer_mods/Lukas/SSD1306_Case_Front_Cover/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This is a front cover replacement that uses a SSD1306 Display.
+
+![](./images/render.PNG)
+
+The holes are the same size in the skirt so you can mount the G-Code Buttons by meteyou https://github.com/VoronDesign/VoronUsers/tree/master/legacy_printers/printer_mods/meteyou/gcode_buttons
+
+Notes:  
+I didn't edit the backplate of the original design. I just used some side cutters and cut it out by hand.  
+It mounts the same way as the original using the inserts that melt into it.
```
Comment (newphreak) [inline printer_mods/Lukas/SSD1306_Case_Front_Cover/README.md:11]: using heatset inserts*

File: printer_mods/README.md [hunk 2]
```diff
@@ -250,6 +250,7 @@ like so:
 ||[Trident Inverted Electronics](./LoganFraser/TridentInvertedElectronics) | Flip the Trident electronics bay for top access.  |:x:|:x:|:x:|:x:|:heavy_check_mark:|
 ||[WagoMounts](./LoganFraser/WagoMounts) | Wago Mounts for Din Rail or 2020 Extrusion mounting. |:grey_question:|:grey_question:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 |Long| [Mantis Dual 5015 Toolhead](./Long/Mantis_Dual_5015) | MGN12 Dual 5015 Toolhead for 2.4 & 1.8 |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
+| Lukas | [SSD1306 Case Front Cover](./Lukas/SSD1306_Case_Front_Cover) | SSD1306 Mount replacement for Mini12864 | :x: | :x: | :heavy_check_mark: | :x: | :heavy_check_mark: |
```
Comment (newphreak) [inline printer_mods/README.md:253]: might want to remove the spaces around Lukas like the others. Minor nitpick, doesn't really seem to affect the outcome.

**Review by newphreak — CHANGES_REQUESTED** [review]
Other then this it looks good to go.

**Comment by MapleLeafMakers** [comment]
3mf files aren't permitted in the repository.  can you remove `printer_mods/Lukas/SSD1306_Case_Front_Cover/Voron_SSD1306_Case_Front_Cover.3mf` please?

File: printer_mods/README.md (hunk 2 above)
Comment (Lukas-Sturm) [reply printer_mods/README.md:253]: Yea i left them, seems a bit easier to read when editing :)

**Comment by Lukas-Sturm** [comment]
> 3mf files aren't permitted in the repository. can you remove `printer_mods/Lukas/SSD1306_Case_Front_Cover/Voron_SSD1306_Case_Front_Cover.3mf` please?

Removed it,
what's the reason, I thought they are the future for exchanging models?

File: printer_mods/Lukas/SSD1306_Case_Front_Cover/README.md (hunk 1 above)
Comment (Lukas-Sturm) [reply printer_mods/Lukas/SSD1306_Case_Front_Cover/README.md]: updated

**Comment by MapleLeafMakers** [comment]
> > 3mf files aren't permitted in the repository. can you remove `printer_mods/Lukas/SSD1306_Case_Front_Cover/Voron_SSD1306_Case_Front_Cover.3mf` please?
> 
> Removed it, what's the reason, I thought they are the future for exchanging models?

For now, the best answer I have is that they can cause problems for people who don't know how they work, so as a rule we don't allow them in the repository.

**Comment by bdbell** [comment]
3MF files contain not only the model but also all of the print settings being used by the creator of the 3MF at the time.  Those print settings may be entirely incompatible with the receiving user's printer.

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**